dateutil/utils.py

```diff
@@ -4,6 +4,10 @@
 from datetime import datetime, time
 
 
+class FakeWeakRef:
```

Comment:
`.utils` is a public module and very much *not* the right place for this. It should probably be in `tz._common.py`